spark/src/main/scala/io/ddf/spark/etl/TransformationHandler.scala

```diff
@@ -149,6 +149,15 @@ class TransformationHandler(mDDF: DDF) extends CoreTransformationHandler(mDDF) {
     transformNativeRserve(Array(transformExpression))
   }
 
+  override def transformNativeRserve(transformExpression: String, inPlace: java.lang.Boolean): DDF = {
+    transformNativeRserve(Array(transformExpression), inPlace)
+  }
+
+  override def transformNativeRserve(transformExpression: Array[String], inPlace: java.lang.Boolean): DDF = {
+    val ddf = this.transformNativeRserve(transformExpression)
+    if (inPlace) this.getDDF.updateInplace(ddf) else ddf
+  }
+
   override def transformNativeRserve(transformExpressions: Array[String]): DDF = {
```

Comment:
maybe better to move the implementation into the `transformNativeRserve(transformExpression: Array[String], inPlace: java.lang.Boolean)` function, and make this function to call  the other?
This is deprecated, so it might not be a good idea to make other functions depend on this one.